haiku-apps/masterpiece/patches/masterpiece-r742.patchset

```diff
@@ -0,0 +1,24 @@
+From 30fd464ff84ecd26a5af381853582f623bf8ecf0 Mon Sep 17 00:00:00 2001
+From: Schrijvers Luc <begasus@gmail.com>
+Date: Sat, 29 Jun 2024 18:04:07 +0200
+Subject: Fix launch for release
+
+
+diff --git a/src/CommonFunctions.cpp b/src/CommonFunctions.cpp
+index 60c1d39..f8a4dc2 100644
+--- a/src/CommonFunctions.cpp
++++ b/src/CommonFunctions.cpp
+@@ -287,8 +287,8 @@ SqlObject::SqlObject(sqlite3_stmt* sqlStatement, const char* errorNumber)
+ 	int	sqlValue;
+ 	BString tmpString;
+ 	sqlErrMsg = 0;
+-	BString tmpPath = GetAppDirPath(); // for testing purposes, use local one.
+-	//BString tmpPath = GetUserDirPath(); // for publish purposes, use real location
++	//BString tmpPath = GetAppDirPath(); // for testing purposes, use local one.
++	BString tmpPath = GetUserDirPath(); // for publish purposes, use real location
```

Comment:
@diversys @humdingerb maybe this could be upstreamed? It's needed when packaged, when build from source in Terminal it isn't needed though.

Reply:
Maybe this can be added as a build option? (define). I don't know how this works with makefile_engine though.

Reply:
No idea on that too, I don't see the real reasoning for it anyway :)
Oh, maybe the string above should be enabled too for exporting/publising ... :)